tests/peers_test.go

```diff
@@ -0,0 +1,88 @@
+package tests
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/ethereum/go-ethereum/whisper/whisperv6"
+	"github.com/status-im/status-scale/cluster"
+	"github.com/status-im/status-scale/network"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSentEnvelope(t *testing.T) {
+	// Setup cluster
+	c := ClusterFromConfig()
+	require.NoError(t, c.Create(context.TODO(), cluster.ScaleOpts{Boot: 1, Relay: 2, Deploy: true}))
+	defer c.Clean(context.TODO()) // handle interrupt signal
+
+	var client *rpc.Client
+	// Get the rpc client
+	Eventually(t, func() error {
+		client = c.GetRelay(0).Admin().Client()
```

Comment:
1. you shouldnt need `eventually` for that. client is set only when health check for a node as performed
2. please add a separate method for Client. like c.GetRelay(0).Whsper or Client. ideally to wrap it with useful helper, like  Whisper.DebugPost, Whisper.Post